.travis.yml

```diff
@@ -13,6 +13,7 @@ matrix:
     - php: 7.1
     - php: 7.2
     - php: 7.3
+    - php: 8.0
```

Comment:
Unfortunately, Travis stopped building open source repos.

Reply:
Travis is still running open-source jobs https://www.traviscistatus.com/
They did cut the number of build agents late last year, so there is often a backlog of jobs waiting.

Travis did not seem to start here at all. I cannot see the repo settings, so maybe Travis has got disabled somehow for this repo. A maintainer needs to look at the settings in GitHub and Travis.

I will force a rebase of this PR to see if Travis will trigger now.

Reply:
I haven't seen a job run by Travis in quite a while, and then there's this: https://news.ycombinator.com/item?id=25338983

Anyway, it's not really related to your PR. I just wanted to see if it passes without checking it out locally :)